testsuite/features/support/env.rb

```diff
@@ -98,20 +97,27 @@ def capybara_register_driver
       unhandledPromptBehavior: 'accept'
     )
 
-    Capybara::Selenium::Driver.new(app, browser: :chrome, desired_capabilities: capabilities, http_client: client)
+    Capybara::Selenium::Driver.new(
+      app,
+      browser: :chrome,
+      desired_capabilities: capabilities,
+      http_client: client
+    )
   end
+
+  Selenium::WebDriver.logger.level = :error unless $debug_mode
+  Capybara.default_driver = :headless_chrome
+  Capybara.javascript_driver = :headless_chrome
+  Capybara.default_normalize_ws = true
+  Capybara.enable_aria_label = true
+  Capybara.automatic_label_click = true
+  Capybara.app_host = "https://#{ENV.fetch('SERVER', nil)}"
+  Capybara.server_port = 8888 + ENV['TEST_ENV_NUMBER'].to_i
+  $stdout.puts "Capybara APP Host: #{Capybara.app_host}:#{Capybara.server_port}"
 end
 
+# register chromedriver headless mode
 $capybara_driver = capybara_register_driver
-Selenium::WebDriver.logger.level = :error unless $debug_mode
```

Comment:
Same here

Reply:
It's part of the rest of refactor I was adding to facilitate running tests using parallel_tests gem, but I can revert that part and move it to another branch.

Reply:
Moved to qe-refactor-enabling-parallel-tests branch